docs/pages/pmd/userdocs/suppressing_warnings.md

```diff
@@ -181,6 +181,6 @@ suppress reporting specifically typed parameters which are unused:
 Note for XPath based suppression to work, you must know how to write
 an XPath query that matches the AST structure of the nodes of the
 violations you wish to suppress.  XPath queries are explained in
-[XPath Rule tutorial](/pmd_devdocs_writing_xpath_rules.html).
+[XPath Rule tutorial](/pmd-6.1.0/pmd_userdocs_extending_writing_xpath_rules.html).
```

Comment:
careful here, we shouldn't bind it to the 6.1.0 version of this doc. We can do so avoiding the initial slash of the link (ie: `pmd_userdocs_extending_writing_xpath_rules.html`) or by using the pmd version property (ie: `pmd-{{site.pmd.version}}/pmd_userdocs_extending_writing_xpath_rules.html`)

Reply:
I did replace it by just `/pmd_devdocs_writing_pmd_rules.html` before the merge commit, I think it might suffice

Reply:
It won't suffice. That would link to `https://pmd.github.io/pmd_devdocs_writing_pmd_rules.html` which doesn't exist.

I've just pushed a fix for this and another broken link on this same page.

Reply:
It should if we use a relative link, like `[XPath Rule tutorial](pmd_userdocs_extending_writing_xpath_rules.html)`, shouldn't it?

Reply:
Snapshot deployed by github is available under https://pmd.github.io/pmd/ - snapshot created by the deploy script is available at http://pmd.sourceforge.net/snapshot/ (if sourceforge is alive).

Reply:
Ok thanks! It seems like on the local site built by `bundler exec jekyll serve`, the urls don't have a `/pmd-{{site.pmd.version}}/` segment, so that the link worked. Maybe this could be improved, though I've no idea how to do that. I guess there's one full site living under each `pmd.github.io/pmd-*`?

Reply:
We should use relative links yes, they're easier to read and work on the local site